src/views/Events.vue

```diff
@@ -10,7 +10,7 @@
         <list-card
           v-for="event in futureEvents"
           :key="event.id"
-          :title="event.title"
+          :title="event.name"
```

Comment:
If you're going to change the event to name then you should probably change this attribute name as well